### PR TITLE
fix: [CDS-40187]: fixed trend for failed executions rate

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
#### Screenshots

https://user-images.githubusercontent.com/98325173/180857745-84851567-f871-459a-b334-6a8f05263717.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

File: src/modules/75-cd/pages/dashboard/CDDashboardPage.module.scss [hunk 1]
```diff
@@ -230,3 +230,8 @@ $header-height: 80px;
     color: var(--grey-500);
   }
 }
+
+.ctn {
+  display: 'flex';
```
Comment (rameshx) [inline src/modules/75-cd/pages/dashboard/CDDashboardPage.module.scss:235]:
```suggestion
  display: flex;
```

**Review by rameshx — APPROVED** [review]

File: src/modules/75-cd/pages/dashboard/CDDashboardPage.module.scss (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/pages/dashboard/CDDashboardPage.module.scss]: Thanks,  I missed this one.

**Review by rameshx — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

**Pull request merged**